src/modules/taskList.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable linebreak-style */
+import { taskCompleted, taskEdit, updateValue } from './events.js';
+
+class ToDoList {
+  constructor() {
+    this.taskList = [];
+  }
+
+  createTask(description) {
+    const taskContainer = document.querySelector('.list');
+    const index = taskContainer.children.length + 1;
+    const li = document.createElement('li');
+    li.className = 'task';
+    const checkBox = document.createElement('input');
+    checkBox.type = 'checkbox';
+    checkBox.className = 'checkbox';
+    checkBox.addEventListener('change', (event) => taskCompleted(event, this));
+    const desc = document.createElement('input');
+    desc.className = 'description';
+    desc.value = description;
+    desc.disabled = true;
+    desc.required = true;
```

Comment:
- [ ] Please According to the requirement you should create an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\].  then 

eg: 
```
const array = [
  {
    description: '...',
    completed: ...,
    index: ..,
  },
];
```
- [ ] Implement a function that iterates over the tasks array and populates the HTML list item element for each task. without forgetting that On page load, the task array is rendered dynamically and appear tasks in order of the `index` values for each task.

```
const populatefunction = (...) => {
  const innerHTML = `
      ... `;
 
  parentElement.appendChild(...);
};
```


- [ ] Kindly Read correctly the requirement and do what they asked to do.